### DISCUSSION TITLE
Does WebView2 support to Windows 10 IoT Enterprise LTSC x64 2021?

### DISCUSSION THREAD

**Discussion by vutvps4** [MicrosoftEdge/WebView2Feedback#4952] [Q&A]
WebView2 apps can run on the following versions of Windows:

Windows 11
Windows 10
Windows 10 IoT Enterprise LTSC x32 2019
Windows 10 IoT Enterprise LTSC x64 2019
Windows 10 IoT Enterprise 21h1 x64
Windows Server 2022
Windows Server 2019
Windows Server 2016

Does WebView2 support to Windows 10 IoT Enterprise LTSC x64 2021?

**Comment by sivMSFT** [comment]
Hi @vutvps4, currently Webview2 doesn't support.
